roles/splunk_common/tasks/main.yml

```diff
@@ -40,6 +40,11 @@
   when:
     - java_version is defined and java_version
 
+- include_tasks: set_general_symmkey_password.yml
+  when:
+    - "'pass4SymmKey' in splunk"
```

Comment:
I'd also add an additional check to make sure splunk.pass4SymmKey is not null/empty so we don't needlessly call this play.